recipes/stop_unfocused.lua

```diff
@@ -0,0 +1,118 @@
+--- Manage sending SIGSTOP signals to special clients when they are unfocused.
+--
+-- Clients with ontop = true are ignored.
+--
+-- You can add a client to sigstop_unfocus.ignore_clients to ignore it:
+--
+--     stop_unfocused.ignore_clients[c] = true
+--
+-- To not ignore it anymore, unset it:
+--
+--     stop_unfocused.ignore_clients[c] = nil
+
+local awful = require('awful')
+local timer = require('gears.timer')
+
+local stop_unfocused = {}
+
+-- TODO: do not stop "Sending Message" client window with Thunderbird.
+--       class: Thunderbird, name: Sending Message, instance: Dialog
+stop_unfocused.config = {
+  rules = {
+    {rule = {class = 'qutebrowser'}},
+    {rule = {class = 'Firefox'}},
+    {rule = {class = 'Thunderbird'}},
+  },
+  stop_timeout = 10,
+}
+
+stop_unfocused.ignore_clients = {}
+
+-- Active timers.
+local sigstop_timers = {}
+
+local sigstopped_clients = {}
+function stop_unfocused.sigstop(c)
+  if c.pid then
+    awesome.kill(c.pid, 19)
+    awful.spawn('pkill -STOP -g ' .. c.pid)
```

Comment:
Of course, WebKit uses child processes nowadays (to everyone reading this: And if your webkit version does not, you might have quite an ancient version of a browser engine, which is never a good sign).